js/CRM_Profile_Form_Edit.js

```diff
@@ -54,18 +54,17 @@
       emaildoubleSetError(true);
       return false;
     }
-  }
+  };
 
   /**
    * Do actual comparison of field values.
    * @returns Boolean
    */
   var emaildoubleIsValid = function emaildoubleIsValid() {
     return (
-      CRM.$('input#email-Primary-emaildouble').val()
-      && (CRM.$('input#email-Primary').val() == CRM.$('input#email-Primary-emaildouble').val())
+      CRM.$('input#email-Primary-emaildouble').val() && (CRM.$('input#email-Primary').val() == CRM.$('input#email-Primary-emaildouble').val())
```

Comment:
Please try and fix this by putting `&&` at the end of line 65, and keep this on two lines instead of all on one line.